### ISSUE TITLE
Package submission suggestion

### ISSUE THREAD

**Issue by denelon** [microsoft/winget-cli#3327]
### Description of the new feature / enhancement

I'd like to help users keep winget-pkgs updated.

@simonhollingshead suggested:
* https://github.com/microsoft/winget-pkgs/issues/108254

WinGet could detect when a user has a newer version of a package.
Similar to:
* #870

This might be a new command, or an argument (with corollary settings).

Both `winget list` and `winget upgrade` already evaluate installed packages. When the user has a newer version of a package than the community repository, winget could provide output to request a contribution.

We would certainly need a specification for the next part of the process where a user could be logically prompted and taken through an experience similar to `wingetcreate update --version <version> <PackageIdentifier>` where they provide the URL(s) and help keep the WinGet community repository updated.

**Comment by JustMyGithub** [comment]
I don't think it is a good idea that "random user" creates a manifest and provides a url. Too much potential for misuse.
Rather just report the higher local version to the server and if a version has been reported more than n times, a maintainer will look into this

https://github.com/microsoft/winget-cli/issues/3361

Funny thing though: I had this very similar independently just 1 week later than you :D
